index.html

```diff
@@ -5,9 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>BC Business Registry</title>
-    <link rel="icon" href="<%= BASE_URL %>favicon.png">
-    <link rel="stylesheet" href="<%= BASE_URL %>css/addresscomplete-2.30.min.css">
-    <script type="text/javascript" src="<%= BASE_URL %>js/addresscomplete-2.30.min.js" defer></script>
+    <link rel="icon" href="/favicon.ico">
+    <link rel="stylesheet" href="/css/addresscomplete-2.30.min.css">
+    <script type="text/javascript" src="/js/addresscomplete-2.30.min.js" defer></script>
```

Comment:
This script is loading correctly and working perfectly in local.

However, in the temporary link, I'm getting an error:
![addresscomplete not initialized](https://github.com/bcgov/business-create-ui/assets/122301442/60ae043d-b49e-4605-aa4e-ff46cbf78cbb)

The script is loading. The problem is the environment variable `VUE_APP_ADDRESS_COMPLETE_KEY`. I did some testing here and apparently the env variable is being blank (empty string). It's not being set to anything.

Maybe it doesn't work in the temporary URL? I'm hoping it does get set in DEV.

Reply:
The vaults file looks OK.

Maybe @pwei1018 needs to configure something for the temporary builds.

Reply:
Let's resolve this conversation but keep an item open to test AddressComplete in Dev.

And maybe ping Patrick about the temp build keys.

Reply:
Sounds good.